src/pages/loading-waiting-states/progress-bar/index.tsx

```diff
@@ -104,36 +99,34 @@ const useStyles = makeStyles((theme: Theme) => ({
             width: '100%',
         },
     },
-    infoList: {
+    fileUploadItem: {
         marginBottom: theme.spacing(2),
         '&:last-child': {
             marginBottom: 0,
         },
     },
-    subTitle: {
-        color: Colors.black[200],
-    },
-    snackbar: {
+    snackbarRoot: {
         position: 'inherit',
         transform: 'none',
+        backgroundColor: Colors.black[900],
```

Comment:
Remove this style, and add the `Colors.black[900] style` to `<InfoListItem />` using a `backgroundColor` prop

Reply:
why? The ILI default color is transparent

Reply:
The animation is wrong in the current implementation. There is a subtle shrink animation that didn't style through.

Reply:
I tried to change timeout prop from 3000 to 100 not sure weather it has correct or not. Even tried https://material-ui.com/components/transitions/ fade and grow transitions but it is not giving proper animation. Any suggestions?

Reply:
I see what you mean...the animation is on the content, not on the entire snackbar, so moving the background color makes sense.

I think the animation looks correct the way you have it now Ekta (you might just want to adjust the timeout prop on the transition to 300 instead of 100. 300 is the default animation duration in MUI).